vertx-consul-client/src/main/java/io/vertx/ext/consul/ConsulClient.java

```diff
@@ -609,6 +609,19 @@ static ConsulClient create(Vertx vertx, JsonObject config) {
   ConsulClient infoSession(String id, Handler<AsyncResult<Session>> resultHandler);
 
   /**
+   * Returns the requested session information
+   * This is blocking query unlike {@link ConsulClient#infoSession(String, Handler)}
```

Comment:
Can you explain what's a blocking query. From the Vert.x POV it's not as you pass a handler.